### PR TITLE
DIT-947 Case Status Lozenge Colouring

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/CaseStatusViewSpec.scala [hunk 1]
```diff
@@ -27,11 +27,24 @@ class CaseStatusViewSpec extends ViewSpec {
 
   "Case Status" should {
 
+    "render CANCELLED without the reason code" in {
```
Comment (SteveSugden) [inline test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/CaseStatusViewSpec.scala]: Is this just to test the logic?  Cancelled should always have a reason.

Comment (ed0906) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/CaseStatusViewSpec.scala]: Yeh but from a data model perspective it isn't essential, and we still want it to show in Grey

Comment (ed0906) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/CaseStatusViewSpec.scala]: It just saves Testers any headaches from changing the status in the DB and wondering why it still shows in blue

**Comment by ed0906** [comment]
> Looks good. SomeCaseStatusViewSpec tests that the blue/gray styles are applied for the right status?

I've added some new assertions for the class :)

**Pull request merged**